tests/ml_commons/test_search_task.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+from opensearch_py_ml.ml_commons import MLCommonClient
+from tests import OPENSEARCH_TEST_CLIENT
+
+ml_client = MLCommonClient(OPENSEARCH_TEST_CLIENT)
+
+
+def test_search_task():
```

Comment:
I think in this case test needs to be more detailed. Right now we are just testing if we get 200 status or not. But what if for a bad query we didn't find any result what we supposed to but still we get 200 status, that will not satisfy the requirement of the test. 

So, I would suggest to do test from the perspective of what we are expecting to get with this search result and what is in reality. To do that even if we need to ingest some model or task, we need to do that to get the search result.

Reply:
Ok, I got it. I will try to upload, load model. Then I will try to search based on new task.

Reply:
we are already uploading model and loading models, can we try to use that?

Reply:
As I understood, you mean I should use upload, load stuff from test_ml_commons file.

Reply:
Yeah, for example, when a model is uploaded in the`test_ml_commons` file, a task will be created. We get the task id.

Now we can also search using this search task api to get the same task id, then we can assert that both task ids are same. That can be a good testing that search task api is really providing results.

What do you think?

Reply:
Yeah, good idea. I thought about it, but I wrote the test in another file because I thought that search task is not intended to be in function with upload, load stuff (because purpose is different).

Reply:
Just to warn, I have raised PR for integrating search_model also. It has pretty similar code. I will change test code for search model also.